src/io/inst_vector.h

```diff
@@ -12,6 +12,7 @@
 #include <mxnet/base.h>
 #include <dmlc/base.h>
 #include <mshadow/tensor.h>
+#include <mshadow/tensor_blob.h>
```

Comment:
@piiswrong Did you also mean the same thing? Changing that line to mxnet/tensor_blob.h doesn't compile as well.